CS483/CS483/Kartaclysm/Components/ComponentTrack.cpp

```diff
@@ -503,8 +523,23 @@ namespace Kartaclysm
 				HeatStroke::EasyXML::GetRequiredFloatAttribute(pNodeElement, "z", node.z);
 				HeatStroke::EasyXML::GetRequiredFloatAttribute(pNodeElement, "variation", node.variation);
 				HeatStroke::EasyXML::GetRequiredFloatAttribute(pNodeElement, "radius", node.radius);
+
+				std::string sTriggerIndex;
+				std::string sDefault = ""; // yup
+				HeatStroke::EasyXML::GetOptionalStringAttribute(pNodeElement, "trigger", sTriggerIndex, sDefault);
```

Comment:
Just set sTriggerIndex = "" as default and pass it in for the second parameter as well. Should also use str as prefix